lib/tasks/tag_tasks.rake

```diff
@@ -9,36 +9,36 @@ namespace :Tag do
   end
 
   desc "Reset tag count"
-  task(:reset_count => :environment) do
+  task(reset_count: :environment) do
     Tag.find(:all).each do |t|
       t.taggings_count
     end
     puts "Tag count reset."
   end
 
   desc "Reset taggings count for obviously wrong taggings_count"
-  task(:fix_taggings_count => :environment) do
-    tag_scope = Tag.where("taggings_count < 0")
+  task(fix_taggings_count: :environment) do
+    tag_scope = Tag.where("taggings_count_cache < 0")
     tag_count = tag_scope.count
     tag_scope.each_with_index do |tag, index|
       puts "#{index} / #{tag_count}"
-      t.taggings_count
+      tag.taggings_count
```

Comment:
Ah, okay! That explains why we didn't notice. Is this task something we should remove, or do you want to keep it for a while just in case?